backend/src/v5/models/modelSettings.js

```diff
@@ -51,8 +57,6 @@ Models.deleteModel = async (ts, model) => {
 	if (deletedCount === 0) {
 		throw templates.modelNotFound;
 	}
-
```

Comment:
not fond of how we're publishing the model related events here apart from this one.

Can do we a findOneAndDelete instead so you can figure out if it's a federation and trigger the event here (assuming this is the reason why you moved it out)